test/blackbox-tests/test-cases/describe.t

```diff
@@ -43,6 +43,30 @@ Setup
   >   (name main3)
   >   (libraries cmdliner)
   >   (modules main3))
+  > 
+  > (library
+  >  (name per_module_pp_lib)
+  >  (modules pp1 pp2)
+  >  (preprocess (per_module ((pps ppx_inline_test) pp2)))
```

Comment:
The purpose of the test was precisely to use a ppx (as opposed to an action, which is tested by other tests).
Moreover, some other tests for ``dune describe`` already use the ``inline_test`` ppx.
Is there some "dummy" ppx that I could use, that you would recommend?

Reply:
Search for a `(kind ppx_rewriter)` in the test suite and you will see a few examples of "dummy" rewriters. It's not optimal because it incurs a dependency on ppxlib anyway, but it's better than ppx_inline_test at least.

Reply:
Thanks for the hint. Done in the last commits.